src/disco/components/AddonCompatibilityError/style.scss

```diff
@@ -0,0 +1,16 @@
+@import "~ui/css/vars";
+
+.AddonCompatibilityError {
+  background: $report-base-color;
+  font-size: 12px;
+  margin: 0;
+  padding: 5px;
+  text-align: center;
+  width: 100%;
+
+  &,
+  a,
+  a:link {
```

Comment:
Any other states needed e.g: visited, hover, focus, active  etc or are the defaults ok?